api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -116,35 +117,42 @@ class OidcAuthenticationService {
     if (!response.isSuccessful) {
       const message = 'Une erreur est survenue en récupérant les informations des utilisateurs.';
```

Comment:
Ici ne sont récupéré que les informations pour 1 utilisateur et pour être cohérent avec les autres messages on gagnerait à modifier le message d'erreur :

```suggestion
      const message = 'Une erreur est survenue en récupérant les informations utilisateur.';
```

Reply:
Je fais la modif